### PR TITLE
Base for Client page

### PR DESCRIPTION
**Pull request by gregmaxin**

### CODE REVIEW

File: src/components/Header/Menu/Menu.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default class Menu extends Component {
                             <li><a href="/saved">View Saved Times</a></li>
                             <li><a href="/login">Switch User</a></li>
                             <li><a href="/help">Help</a></li>
+                            <li><a href="/client">Client</a></li>
```
Comment (Ruxuan) [inline src/components/Header/Menu/Menu.js:27]: From my understanding, Clients page is for families/patients. Do we want to connect it with the CSA side like this? Or is this just for testing?

File: src/App.js [hunk 2]
```diff
@@ -250,8 +251,14 @@ export default class App extends Component {
...
 
-        var db = firebase.firestore();
+        const ClientPage = (props) => {
+            return (
+                <Client
+                />
+            )
+        }
 
+        var db = firebase.firestore();
```
Comment (Ruxuan) [inline src/App.js:310]: Does `const` or `let` work here?

**Review by Ruxuan — APPROVED** [review]
looks good. It seems like we should agree on a tabs convention as well.

File: src/App.js (hunk 2 above)
Comment (gregmaxin) [reply src/App.js:310]: That is a good question - I'm not sure. 

I think `let` would also work here

File: src/components/Header/Menu/Menu.js (hunk 1 above)
Comment (gregmaxin) [reply src/components/Header/Menu/Menu.js:27]: For now, it is just for testing - It will be split up later. 

Your understanding is correct.

**Pull request merged**